services/tenant-ui/frontend/src/components/profile/issuance/EndorserConnect.vue

```diff
@@ -1,39 +1,70 @@
 <template>
-  <div v-if="showEndorserConnect">
-    <Button
-      title="Connect to endorser"
-      icon="pi pi-user-plus"
-      class="p-button-rounded p-button-icon-only p-button-text"
-      @click="connectToLedger()"
-    />
-  </div>
-
-  <div v-if="showLedgerSwitch">
-    <Button
-      label="Switch Ledger"
-      icon="pi pi-arrow-right-arrow-left"
-      text
-      @click="switchLedger($event)"
-    />
-  </div>
-
-  <div
-    v-if="endorserConnection && props.ledgerInfo.ledger_id === currWriteLedger"
-    class="flex"
-  >
-    <div class="flex align-items-center mr-2">{{ $t('common.status') }}</div>
-    <div class="flex align-items-center mr-1">
-      <StatusChip :status="endorserConnection.state" />
+  <template v-if="isWebvhLedger">
+    <div v-if="showEndorserConnect">
+      <Button
+        title="Connect to webvh endorser"
+        icon="pi pi-user-plus"
+        class="p-button-rounded p-button-icon-only p-button-text"
+        @click="connectToLedger()"
+      />
+    </div>
+    <div v-else-if="webvhConnection" class="flex">
+      <div class="flex align-items-center mr-2">{{ $t('common.status') }}</div>
+      <div class="flex align-items-center mr-1">
+        <StatusChip :status="webvhConnection.state || 'unknown'" />
+      </div>
+      <div v-if="canDeleteConnection" class="flex align-items-center">
+        <Button
+          title="Delete Connection"
+          icon="pi pi-trash"
+          class="p-button-rounded p-button-icon-only p-button-text"
+          @click="deleteConnection($event, webvhConnection.connection_id)"
+        />
+      </div>
     </div>
-    <div v-if="canDeleteConnection" class="flex align-items-center">
+    <div v-else class="flex align-items-center">
+      <i class="pi pi-spin pi-spinner" style="font-size: 1.5rem"></i>
+    </div>
+  </template>
+  <template v-else>
+    <div v-if="showEndorserConnect">
       <Button
-        title="Delete Connection"
-        icon="pi pi-trash"
+        title="Connect to endorser"
+        icon="pi pi-user-plus"
         class="p-button-rounded p-button-icon-only p-button-text"
-        @click="deleteConnection($event, endorserConnection.connection_id)"
+        @click="connectToLedger()"
+      />
+    </div>
+
+    <div v-if="showLedgerSwitch">
+      <Button
+        label="Switch Ledger"
+        icon="pi pi-arrow-right-arrow-left"
+        text
+        @click="switchLedger($event)"
       />
     </div>
-  </div>
+
+    <div
+      v-if="
+        endorserConnection && props.ledgerInfo.ledger_id === currWriteLedger
+      "
+      class="flex"
+    >
+      <div class="flex align-items-center mr-2">{{ $t('common.status') }}</div>
+      <div class="flex align-items-center mr-1">
+        <StatusChip :status="endorserConnection.state" />
+      </div>
+      <div v-if="canDeleteConnection" class="flex align-items-center">
```

Comment:
Think something has flipped with the canDeleteConnectionLogic.

On existing version once I connect to a Ledger I can't delete a connection once I've completed onboarding:

<img width="1086" height="573" alt="Image" src="https://github.com/user-attachments/assets/eaabe007-7053-42ac-88d5-46b321048340" />


but on this version I can:

<img width="737" height="374" alt="Image" src="https://github.com/user-attachments/assets/2a426780-5647-447b-b99c-a51f83f82738" />

Intended change?